dashboard/src/components/CommentsGeneric.js

```diff
@@ -539,7 +539,7 @@ function CommentModal({
                         name="comment"
                         placeholder="Tapez votre commentaire ici..."
                         value={values.comment || ''}
-                        rows={3}
+                        rows={5}
```

Comment:
allons-y carrément même, pourquoi pas !

```suggestion
                        rows={10}
```

Reply:
Pas sûr pour les plus petits écrans (déjà le mien c'est pas fou). Comme ça se redimensionne, c'est déjà bien.

Reply:
Bon j'ai mis 7